src/utils/CachingUtils.ts

```diff
@@ -44,10 +44,10 @@ export async function setDepositInCache(
 }
 
 /**
- * Resolves the key for caching either a deposit or a fill.
- * @param depositOrFill Either a deposit or a fill. In either case, the depositId and originChainId are used to generate the key.
- * @returns The key for caching the deposit or fill.
+ * Resolves the key for caching a bridge event.
+ * @param bridgeEvent The depositId and originChainId are used to generate the key.
+ * @returns The key for caching the event.
  */
-export function getDepositKey(depositOrFill: Deposit | Fill): string {
-  return `deposit_${depositOrFill.originChainId}_${depositOrFill.depositId}`;
+export function getDepositKey(bridgeEvent: Deposit | Fill | SlowFillRequest): string {
```

Comment:
Is it OK to use this for SlowFillRequest objects? In the SpokePoolClient I concluded that we needed a full RelayData hash, because SlowFillRequest permits the user to control _all_ of the fields that would be used in a lookup (i.e. the `depositId` and `originChainId`). So it's very easy for someone to cause confusion in the bots. By using a RelayData hash then we should also get the depositor included, so it's not possible for them to spoof or confuse the bots about a separate event.

Reply:
I thnk a better fix for this is to switch on the version of the event and either use the existing key or the new getV3Relayhash to create a unique cache key: https://github.com/across-protocol/sdk-v2/pull/535/commits/7ed442207538be35ec264567085467efe6684e50